main.cpp

```diff
@@ -77,45 +97,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 } 
 
 /*
- 1)
+ 1) 
  */
-
+bool rentHotelRoom(int rentalLength, int suiteType = 2)
+{
+    ignoreUnused(rentalLength,suiteType);
+    return { };
+}
 /*
  2)
  */
-
+bool cookDinner(int ingredientsList, int setTable, bool candleLit = true)
+{
+    ignoreUnused(ingredientsList, setTable);
+    return { };
+}
 /*
  3)
  */
-
+bool isInTune(char tuning = 'A')
+{
+    ignoreUnused(tuning);
+    return { };
+}
 /*
  4)
  */
-
+int battingAverage(float leadOff = 0.290f, float cleanUp = 0.305f)
```

Comment:
choose a function name that contains a verb.
`battingAverage` is a property that can be represented with a primitive type. 
- [ ] revise

Reply:
Not sure if `battingAverage is a property that can be represented with a primitive type.` means you are asking me to change something besides adding a verb to the function name.